tests/kernel/common/testcase.yaml

```diff
@@ -23,6 +23,7 @@ tests:
     extra_configs:
       - CONFIG_CBPRINTF_NANO=y
       - CONFIG_CBPRINTF_REDUCED_INTEGRAL=y
+    platform_exclude: qemu_arc_hs6x
```

Comment:
Does this still fail, merged upstream fixes just recently.

Reply:
There is an issue with printf subtest on ARC64 qemu platform so I exclude test for now. It works fine on ARC64 nsim simulation platform so it could be ARC64 qemu issue itself.